assets/components/emailFormField/emailFormField.jsx

```diff
@@ -4,33 +4,58 @@
 
 import React from 'react';
 import TextInput from 'components/textInput/textInput';
+import ErrorMessage from 'components/errorMessage/errorMessage';
+import { validateEmailAddress } from '../../helpers/utilities';
 
 // ----- Types ----- //
 
 type PropTypes = {
   emailUpdate: (email: string) => void,
   email: string,
   isSignedIn: boolean,
+  setEmailHasBeenBlurred: () => void,
+  emailHasBeenBlurred: boolean,
 };
 
-
 // ----- Component ----- //
 
 const EmailFormField = (props: PropTypes) => {
 
+  const showEmailError = props.emailHasBeenBlurred && !validateEmailAddress(props.email);
+
   if (props.isSignedIn) {
     return null;
   }
 
-  return (<TextInput
-    id="email"
-    value={props.email}
-    labelText="Email"
-    placeholder="Email"
-    onChange={props.emailUpdate}
-    modifierClasses={['email']}
-    required
-  />);
+  const modifierClass = ['email'];
+
+  if (showEmailError) {
+    modifierClass.push('error');
+  }
+
+  const errorMessage = () => {
+    if (!showEmailError) {
+      return '';
+    }
+    return (<ErrorMessage message="Please enter a valid email address." />);
+  };
```

Comment:
This would work as an aux component:

```jsx
function EmailError(props: { showError: boolean }) {

  if (!showError) {
    return null;
  }

  return <ErrorMessage message="Please enter a valid email address." />;

}
```

The conventional way to [render nothing](https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering) is with `null`.